src/main/java/com/eleks/academy/whoami/core/state/WaitingForPlayers.java

```diff
@@ -42,4 +43,15 @@ public int getPlayersInGame() {
 	public GameStatus getStatus() {
 		return WAITING_FOR_PLAYERS;
 	}
+
+	@Override
+	public List<SynchronousPlayer> getPlayersList() {
```

Comment:
Much better is to have a getter that just returns `Map<String, SynchronousPlayer> players` and in a place where it is used to have code that transforms it into needed value. So just have a getter and remove this method from interface

Reply:
Resolve